### PR TITLE
Re-link themis as external lib

### PR DESCRIPTION
**Pull request by vixentael**
Hi guys, 

Lately Themis was uploaded to bintray, so now it can be linked as external library, instead of `.aar` file.

```
repositories {
    google()
    jcenter()
    maven { url "https://dl.bintray.com/cossacklabs/maven/" }
}

dependencies {
    implementation 'com.cossacklabs.com:themis:0.10.0'
}
```

I was trying to propagate this change into Android-Secure-Storage, but I'm not sure that handled everything correctly, especially the distribution phase.

Please review, and update :)

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -34,9 +34,6 @@ publishing {
             groupId packageName
             version = libraryVersion
             artifactId project.getName()
-
-            // Tell maven to prepare the generated "*.aar" file for publishing
-            artifact("$buildDir/outputs/aar/${project.getName()}-release.aar")
```
Comment (vixentael) [inline app/build.gradle]: not sure if I should removed this

Comment (vladimir-ivanov-epam) [reply app/build.gradle]: Please, revert this change, it's about publishing the storage libary itself)

File: app/build.gradle [hunk 2]
```diff
@@ -72,5 +69,5 @@ dependencies {
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
 
     //Themis
-    implementation project(path: ':themis')
+    implementation 'com.cossacklabs.com:themis:0.10.0'
```
Comment (vladimir-ivanov-epam) [inline app/build.gradle:75]: Awesome!

**Review by vladimir-ivanov-epam — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (vixentael) [reply app/build.gradle]: that's what i thought!
put it back in latest commit

**Comment by vixentael** [comment]
@vladimir-ivanov-epam 
please take a look one more time :)

**Review by vladimir-ivanov-epam — APPROVED** [review]

**Pull request merged**